content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -100,17 +83,40 @@ writes the model's
 [ROC-AUC](https://scikit-learn.org/stable/modules/model_evaluation.html#receiver-operating-characteristic-roc)
 and
 [average precision](https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-and-f-measures)
-to `evaluation/metrics.json`, which in turn is marked as a `metrics` file with
-`-M`. Its contents are:
+for both train and test datasets to `eval/live/metrics.json`, which in turn is
+marked as a `metrics` file with `-M`. Its contents are:
 
 ```json
-{ "avg_prec": 0.5204838673030754, "roc_auc": 0.9032012604172255 }
+{
+  "avg_prec": {
+    "train": 0.9772271756725741,
+    "test": 0.9449556493816984
+  },
+  "roc_auc": {
+    "train": 0.9873675866013153,
+    "test": 0.9619097316125981
+  }
+}
 ```
 
+> DVC doesn't force you to use any specific file names, nor does it enforce a
+> format or structure of a metrics file. It's completely user/case-defined.
+> Refer to `dvc metrics` for more details.
+
+You can view tracked metrics with DVC:
+
+```dvc
+$ dvc metrics show
+Path                    avg_prec.test    avg_prec.train    roc_auc.test    roc_auc.train
+eval/live/metrics.json  0.94496          0.97723           0.96191         0.98737
+```
+
+## Configuring plots
+
 `evaluate.py` also writes `precision`, `recall`, and `thresholds` arrays
 (obtained using
 [`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
-into the plots file `evaluation/plots/prc.json`:
+into the plots files `eval/prc/train.json` and `eval/prc/test.json`:
 
 ```json
```

Comment:
Q: Which one is seen in the sample below?

Reply:
TBH I don't know. It's only an example of the file structure. Does it matter whether it shows train or test metrics?

Reply:
No. I just made it clearer that it applies to both in the follow-up. Thanks